src/style.css

```diff
@@ -0,0 +1,75 @@
+body {
+  margin: 0;
+}
+
+.conteiner {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.settings-block {
+  display: flex;
+  flex: 1 0 33%;
+  align-items: center;
+  justify-content: center;
+  min-height: 300px;
+  font-size: 100px;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.first-block {
+  background-color: rgb(255, 0, 0);
+}
+
+.second-block {
+  background-color: rgb(205, 0, 0);
+}
+
+.third-block {
+  background-color: rgb(154, 0, 0);
+  order: 3;
```

Comment:
You don't need to provide order property to those elements, which order should be the same as in HTML.